EN feedback widget

rfrenchseti opened the pull request.

rfrenchseti committed:
First pass at NASA feedback widget integration

rfrenchseti committed:
Move feedback to the Help menu. Use the feedbackTab widget.

rfrenchseti committed:
Disable captcha code. Change responsiveness of nav header. Badly hack feedback.js and feedback.css to remove errors and force into OPUS UI style.

juzen2003 reviewed (commented):
Do we want to add the responsiveness to feedback form and welcome message modal?
  - The feedback form starts to get cut off when browser height is about 605px or less. 
  - Same as the welcome modal which starts to get cuff off when browser height is about 575px or less.

rfrenchseti committed:
Clean up feedback CSS

rfrenchseti committed:
Merge remote-tracking branch 'upstream/master' into en_feedback_widget

rfrenchseti committed:
Auto stash before merge of "en_feedback_widget" and "upstream/master"

Feedback widget - add blog link, make feedback only send email for production versions.

rfrenchseti committed:
Add feedback tab close on escape key
